projects/1.18/assets/alcocraft-beer-and-stuff/alcocraft/lang/zh_cn.json

```
@@ -0,0 +1,59 @@
+{
+  "itemGroup.alcotab": "酿酒工艺",
+
+  "block.alcocraft.spruce_keg": "酿酒桶",
+  "block.alcocraft.hop_block": "啤酒花",
+  "block.alcocraft.hop_plant_block": "啤酒花",
+
+  "item.alcocraft.hop": "啤酒花",
+  "item.alcocraft.hop_seeds": "啤酒花种子",
+  "item.alcocraft.dry_seeds": "干小麦种子",
+
+  "item.alcocraft.spruce_mug_empty": "空酒杯",
+
+  "item.alcocraft.spruce_mug_sun_pale_ale": "阳式淡色艾尔啤酒",
+  "item.alcocraft.spruce_mug_digger_bitter": "矿工苦杯",
+  "item.alcocraft.spruce_mug_nether_porter": "下界波特酒",
+  "item.alcocraft.spruce_mug_wither_stout": "凋灵黑酒",
+  "item.alcocraft.spruce_mug_magnet_pilsner": "红石皮尔森",
+  "item.alcocraft.spruce_mug_drowned_ale": "溺水艾尔酒",
+  "item.alcocraft.spruce_mug_night_rauch": "午夜烟熏",
+  "item.alcocraft.spruce_mug_ice_beer": "冰啤",
+  "item.alcocraft.spruce_mug_kvass": "格瓦斯",
+  "item.alcocraft.spruce_mug_leprechaun_cider": "妖精绿啤",
+  "item.alcocraft.spruce_mug_chorus_ale": "紫颂艾尔",
+  "item.alcocraft.spruce_mug_nether_star_lager": "下界星拉格",
+
+  "block.alcocraft.spruce_mug_block_empty": "空酒杯",
+
+  "block.alcocraft.spruce_mug_block_sun_pale_ale": "阳式淡色艾尔啤酒",
+  "block.alcocraft.spruce_mug_block_digger_bitter": "矿工苦杯",
+  "block.alcocraft.spruce_mug_block_nether_porter": "下界波特酒",
+  "block.alcocraft.spruce_mug_block_wither_stout": "凋灵黑酒",
+  "block.alcocraft.spruce_mug_block_magnet_pilsner": "红石皮尔森",
+  "block.alcocraft.spruce_mug_block_drowned_ale": "溺水艾尔酒",
+  "block.alcocraft.spruce_mug_block_night_rauch": "午夜烟熏",
+  "block.alcocraft.spruce_mug_block_ice_beer": "冰啤",
+  "block.alcocraft.spruce_mug_block_kvass": "格瓦斯",
+  "block.alcocraft.spruce_mug_block_leprechaun_cider": "妖精绿啤",
+  "block.alcocraft.spruce_mug_block_chorus_ale": "紫颂艾尔",
+  "block.alcocraft.spruce_mug_block_nether_star_lager": "下界星拉格",
+
+  "effect.alcocraft.flight": "飞行",
+  "effect.alcocraft.freeze": "冰霜",
+  "effect.alcocraft.phantom": "精力充沛",
+  "effect.alcocraft.attract": "吸引",
+  "effect.alcocraft.wither": "凋零之触",
+
+  "advancements.husbandry.hop.title": "欢呼雀跃",
+  "advancements.husbandry.hop.description": "通过在方块底部种植啤酒花种子来获得啤酒花",
+
+  "advancements.husbandry.keg.title": "酿酒之行，始于酒桶",
+  "advancements.husbandry.keg.description": "制作一个酒桶，开始你的酿酒之旅。",
+
+  "advancements.husbandry.mug.title": "喝还是不喝",
+  "advancements.husbandry.mug.description": "酿出你的第一杯啤酒",
+
+  "advancements.husbandry.all_beer.title": "必须把酒全喝一遍",

```

```suggestion
  "advancements.husbandry.all_beer.title": "把它们全喝掉",
```
欧美宝可梦梗，原梗是`Gotta catch em all`，译名见[神奇宝贝百科](https://wiki.52poke.com/wiki/%E6%94%B6%E6%9C%8D%E5%88%B0%E5%AF%B6%E5%8F%AF%E5%A4%A2%E4%BA%86)

---

抱歉，本人从未接触过宝可梦